### ISSUE TITLE
Manually conteolled turret

### ISSUE THREAD

**Issue by weseli51** [Anuken/Mindustry-Suggestions#39]
What about the turret that youcan controll instead of being a ship
You will be much stronger but ammo will be required and you won't be able to move

**Comment by KK6DOC** [comment]
Anuke already got this suggestion and because of the 6.0 mech rework he said this would be very hard to implement

**Comment by MEEPofFaith** [comment]
I mean, there is the Vertex in the [whert-dev/mindustry-whert-tex](https://github.com/whert-dev/mindustry-whert-tex) mod

**Comment by Anuken** [comment]
Not relevant in v6
